dev/devicelab/bin/tasks/drive_perf_debug_warning.dart

```diff
@@ -16,6 +16,7 @@ Future<String> _runWithMode(String mode, String deviceId) async {
     '-d',
     deviceId,
   ]);
+  await evalFlutter('install', options: <String>['--uninstall-only', '-d', deviceId]);
```

Comment:
I don't think this needs to be modified but one difference between this code and the `uninstallApp` code is that the uninstall app code pauses for 2 seconds then awaits the device to come back available (which can take a minute). This only matters on android because all other device types do nothing on awaitDevice.

Reply:
yeah... i considered removing the the second await (it violates our style guide as it is likely to introduce flakes) but decided that it'd be best to do that in a separate PR.